Core/src/OffConverter/OffReader.cpp

```diff
@@ -238,7 +236,14 @@ buw::Vector3f OffReader::calcNormal(const buw::Vector3f& vertex1,
 	carve::geom::vector<3> normal = carve::geom::cross(vector1 - vector2, vector2 - vector3);
 	normal.normalize();
 
-	return buw::Vector3f(normal.x, normal.y, normal.z);
+	return buw::Vector3f(normal.x, normal.y, normal.z);	
+}
+buw::Vector3f OffReader::readColorsFromFace(std::stringstream& lineStream)
+{
+	carve::geom::vector<3> colorVector;
+	lineStream >> colorVector[0] >> colorVector[1] >> colorVector[2];
+
+	return buw::Vector3f(colorVector.x, colorVector.y, colorVector.z);
```

Comment:
To normalize this, divide each number with 255.

Reply:
Resolved bbb7fb7